src/qemu.rs

```diff
@@ -468,21 +468,10 @@ impl Qemu {
     /// Construct a QEMU instance backing a vmtest target.
     ///
     /// Does not run anything yet.
-    pub fn new(
-        updates: Sender<Output>,
-        image: Option<&Path>,
-        kernel: Option<&Path>,
-        kargs: Option<&String>,
-        rootfs: &Path,
-        bios: Option<&Path>,
-        command: &str,
-        host_shared: &Path,
-        uefi: bool,
-        vm: &VMConfig,
-    ) -> Result<Self> {
+    pub fn new(updates: Sender<Output>, target: Target, host_shared: &Path) -> Result<Self> {
```

Comment:
Would be better to pass a `&Target` and let callee clone as necessary. Looks like most of the previous args were references already

Reply:
yes. make sense.